### PR TITLE
[OUI] Removed figma link/icon, Codesandbox link/icon, Sketch link/icon Casc…

### LINKED ISSUE #111: (OUI Docs) Docs Site Header
These changes are necessary before go live: 

- [x] Remove Figma link/icon
- [x] Remove Codesandbox link/icon
- [x] Remove Cascadia options (Cascadia: Light, Cascadia: Dark) from theme dropdown
- [x] Remove Babelfish (comment out pending further testing and localization strategy) 
- [x] Change logoElastic to logoOpenSearch
- [x] Replace favicon with OpenSearch favicon

### PR DESCRIPTION
**Pull request by AbhishekReddy1127**
Signed-off-by: AbhishekReddy1127 <nallamsa@amazon.com>

### Description
Removed Figma, Codesandbox, Sketch link and icon.
Removed Cascadia options and Babelfish from theme dropdown.
 
### Issues Resolved
#111 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by KrooshalUX — APPROVED** [review]
Based on descriptions in the PR, this matches the prescribed changes in 111. LGTM!

File: src-docs/src/components/guide_page/guide_page_header.tsx [hunk 1]
```diff
@@ -128,12 +103,7 @@ export const GuidePageHeader: React.FunctionComponent<GuidePageHeaderProps> = ({
         button={button}
         isOpen={mobilePopoverIsOpen}
         closePopover={() => setMobilePopoverIsOpen(false)}>
-        <div className="guideOptionsPopover">
-          {renderGithub()}
-          <GuideSketchLink />
-          <GuideFigmaLink />
-          {renderCodeSandbox()}
```
Comment (BSFishy) [inline src-docs/src/components/guide_page/guide_page_header.tsx:135]: Are these components used anywhere else? If not, can the components themselves be deleted?

File: src-docs/src/components/guide_theme_selector/guide_theme_selector.tsx [hunk 2]
```diff
@@ -45,8 +42,6 @@ export const GuideThemeSelector: React.FunctionComponent<GuideThemeSelectorProps
 // @ts-ignore Context has no type
 const GuideThemeSelectorComponent: React.FunctionComponent<GuideThemeSelectorProps> = ({
   context,
-  onToggleLocale,
-  selectedLocale,
```
Comment (BSFishy) [inline src-docs/src/components/guide_theme_selector/guide_theme_selector.tsx:49]: If these props aren't being used anymore, can they be removed from the props interface?

**Review by BSFishy — CHANGES_REQUESTED** [review]

**Review by BSFishy — APPROVED** [review]

**Review by ashwin-pc — COMMENTED** [review]
Why was the call made to remove locale selection?

**Comment by KrooshalUX** [comment]
@ashwin-pc  because we cannot support it right now - please see the original task in the issue #111 was to "Remove Babelfish (comment out pending further testing and localization strategy)"

**Pull request merged**